schedule/index.html

```diff
@@ -123,6 +123,9 @@
                 <li><a href="#2016-03-20-exhibition">Exhibition</a></li>
               </ul>
           </ul>
+          <form class="pull-right" style="padding:1%">
```

Comment:
Do not edit directly, submit a PR to git@github.com:fossasia/open-event-scraper.git